backend/ee/danswer/db/document.py

```diff
@@ -3,12 +3,61 @@
 from sqlalchemy import select
 from sqlalchemy.orm import Session
 
-from danswer.db.models import Document
+from danswer.access.models import ExternalAccess
+from danswer.access.utils import prefix_group_w_source
+from danswer.configs.constants import DocumentSource
+from danswer.db.models import Document as DbDocument
 
 
 def fetch_documents_from_ids(
```

Comment:
get_documents_by_ids <- there's this version in the MIT that maybe we can just give an arg to make it throw an exception possibly? less duplicated code, but totally nit